estimators/ccb/first_slot.py

```
@@ -0,0 +1,39 @@
+from estimators.ccb import base
+
+class Estimator(base.Estimator):
+    def __init__(self, bandits_estimator):
+        self.estimator = bandits_estimator
+
+    def add_example(self, p_logs, r, p_preds, count=1):
+        """Expects lists for logged probabilities, rewards and predicted probabilities. These should correspond to each slot."""
+
+        if not isinstance(p_logs, list) and not isinstance(r, list) and not isinstance(p_preds, list):
+            raise('Error: p_logs, r and p_preds must be lists')
```

Can you raise a specific error type such as `ValueError`